src/common/Microsoft.DotNet.UpgradeAssistant.Abstractions/NuGetReference.cs

```diff
@@ -7,7 +7,7 @@ namespace Microsoft.DotNet.UpgradeAssistant
 {
     public record NuGetReference(string Name, string Version)
     {
-        public bool HasWildcardVersion => Version.Equals("*", StringComparison.OrdinalIgnoreCase);
+        public bool HasWildcardVersion => Version.Contains("*");
```

Comment:
Can you add a comment as the forms wildcard version can take?

Reply:
added